kolibri/utils/debian_check.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+
+from builtins import input
+
+
+def check_debian_user():
+    # Check whether the current user is the kolibri user when running kolibri
+    # that is installed from .deb package.
+    # The code is mainly from https://github.com/learningequality/ka-lite/blob/master/bin/kalite#L53
+    if os.name == "posix" and os.path.isfile("/etc/kolibri/username"):
+        kolibri_user = open("/etc/kolibri/username", "r").read().rstrip()
```

Comment:
should we maybe wrap this in a `with`?

```python
with open("/etc/kolibri/username", "r") as f:
    kolibri_user = f.read().rstrip()
```

Reply:
yes! thank you for the suggestion!

Reply:
FYI, garbage collection closes file objects automatically :)